src/pages/Schedule.js

```diff
@@ -0,0 +1,314 @@
+                </EventNoDate>
+            </EventGroup>
+
+            <div css={`text-align: center;
+                        border-style: solid;
+                        border-radius: 40px;
+                        width: 70vw;
+                        padding: 3vw;
+                        display: block;
+                        margin: 4vw auto 0`}>
+                <Text fontSize="2rem" roboto bold>First Round Judging <br/> Due 4/16 at 11:59 PM EST</Text><br/>
+                <Button href="https://tinyurl.com/PCVPrizeSubmission">Submit Your Decks!</Button>
+            </div>
+        </div>
+    )
+}
+
+const Finals = () => {
+    return (
+        <div css={`text-align: center;
+                   border-style: solid;
+                   border-radius: 40px;
+                   width: 70vw;
+                   padding: 3vw;
+                   display: block;
+                   margin: 4vw auto`}>
+            <Text fontSize="2rem" roboto bold>Live Finals <br/> 4/24 at 12 PM EST <br/> with keynote from</Text>
+            <div css={`display: flex;
+                        justify-content: center;
+                        align-items: center;
+                        padding: 2vw;`}>
+                <Headshot src={Dennis} alt="Dennis Woodside"/>
+                <Caption css={`text-align: left`}>
+                    <b>Dennis Woodside</b>
+                    <br/>
+                    <Text>President of Impossible Foods</Text>
+                </Caption>
+            </div>
+
+            <Text fontSize="2rem" roboto bold>and judging by</Text>
+
+            <Columns>
+                <figure>
+                    <Headshot src={Lidiya} alt="Lidiya Dervisheva"/>
+                    <Caption>
+                        <b>Lidiya Dervisheva</b>
+                        <br/>
+                        <Text>Partner <br/> @ Next47</Text>
+                    </Caption>
+                </figure>
+                <figure>
+                    <Headshot src={Seth} alt="Seth Bannon"/>
+                    <Caption>
+                        <b>Seth Bannon</b>
+                        <br/>
+                        <Text>Founding Partner <br/>@ Fifty Years</Text>
+                    </Caption>
+                </figure>
+                <figure>
+                    <Headshot src={Marissa} alt="Marisa Sweeney"/>
+                    <Caption>
+                        <b>Marisa Sweeney</b>
+                        <br/>
+                        <Text>Director <br/>@ Generate Capital</Text>
+                    </Caption>
+                </figure>
+                <figure>
+                    <Headshot src={Sam} alt="Sam Bursten"/>
+                    <Caption>
+                        <b>Sam Bursten</b>
+                        <br/>
+                        <Text>Vice President <br/>@ Energy Impact Partners</Text>
+                    </Caption>
+                </figure>
+            </Columns>
```

Comment:
Since these are all formatted the same, might be worth it to extract a Figure component that takes in headshot, name, title and company props.

Reply:
When I say might I think this violates DRY so I mean def : |